common/src/storage.rs

```diff
@@ -58,16 +58,16 @@ impl MemoryKeyValue {
 }
 
 impl KeyValue for MemoryKeyValue {
-    fn get(&self, key: Vec<u8>) -> Fallible<Vec<u8>> {
+    fn get(&self, key: Vec<u8>) -> Result<Vec<u8>> {
         self.0
             .lock()
             .unwrap()
             .get(&key)
             .cloned()
-            .ok_or(format_err!("not found"))
+            .ok_or(anyhow!("not found"))
```

Comment:
you'll want to [ok_or_else](https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or_else) this. [the macro](https://docs.rs/anyhow/1.0.31/src/anyhow/macros.rs.html#153) expands to a non-const fn